be/src/olap/rowset/segment_v2/bloom_filter.h

```diff
@@ -194,7 +194,7 @@ class BloomFilter {
     /// Bloom filters must have equal size and seed.
     virtual bool contains(const BloomFilter& bf_) const { return true; };
 
-    virtual char* data() const { return _data; }
```

Comment:
Bloomfilter structure should be immutable. If is is mutable, I think there are many problems. For example, if we have a bloom filter index, and 2 concurrent queries, if one query thread modify bf, there will be crash.